test/get_previous_releases.py

```diff
@@ -134,6 +134,9 @@ def progress_hook(progress_bytes, total_size):
                 progress_bytes += len(chunk)
                 progress_hook(progress_bytes, total_size)
 
+        if progress_bytes < total_size:
```

Comment:
we could be stricter and make it `!=`

Reply:
Actually, `progress_bytes < total_size:` could be considered stricter, because if the server starts replying with the wrong payload size, or appends data, it seems like this could be useful to know via a checksum error.